grails-app/controllers/au/org/emii/portal/WpsController.groovy

```diff
@@ -63,7 +64,7 @@ class WpsController extends HostVerifyingController {
                 job: [
                     uuid: params.uuid,
                     reportUrl: g.createLink(action: 'jobReport', absolute: true, params: params),
-                    createdTimestamp: new DateTime(String.valueOf(execResponse.Status.@creationTime)),
+                    createdTimestamp: new DateTime(String.valueOf(execResponse.Status.@creationTime)).withZone( DateTimeZone.forID( "Australia/Melbourne" )),
```

Comment:
What happens if a portal-end user is outside australia/melbourne? will setting the timezone explicitly change any display behavior on the client side?